app/models/post.rb

```diff
@@ -12,5 +12,5 @@ def recent_comments
     comments.limit(5).order(created_at: :desc)
   end
 
-  private :update_counter
+  # private :update_counter
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.